src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -121,3 +121,85 @@ end
 
 
 
+
+"""
+    struct ConstantLifetimeChargeTrappingModel{T <: SSDFloat} <: AbstractChargeTrappingModel{T}
+        
+This constant-lifetime-based charge trapping model is similar to the Boggs model, which is constant-mean-free-path based.
```

Comment:
Maybe define this without referencing `Boggs` here, something like
```suggestion
This constant-lifetime-based charge trapping model assumes electrons and holes to have a constant free lifetime throughout the crystal.
```